autogpt_platform/frontend/src/lib/autogpt-server-api/client.ts

```diff
@@ -865,6 +867,10 @@ export default class BackendAPI {
       graph_version: graphVersion,
     });
   }
+
+  async askOtto(query: OttoQuery): Promise<OttoResponse> {
+    return this._request("POST", "/otto/ask", query);
```

Comment:
pull user id here and attach to query

Reply:
the user id should be setn with request actually

Reply:
You don't need to attach the userid here is what I'm saying. it'll be handled automatically